### PR TITLE
ansible: vendor recent fixes

### PR DESCRIPTION
**Pull request by mapuri**
this pulls in the fixes done in https://github.com/contiv/ansible/pull/245 along with other recent updates in the ansible repo

/cc @erikh @dseevr

### CODE REVIEW

**Comment by dseevr** [comment]
LGTM

File: ansible/roles/contiv_storage/defaults/main.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # Role defaults for contiv_storage
 
 contiv_storage_supervisor_host: ""
-contiv_storage_version: "v0.1.0"
+contiv_storage_version: "v0.0.0-06-30-2016.09-52-49.UTC"
```
Comment (erikh) [inline ansible/roles/contiv_storage/defaults/main.yml]: I just removed these releases. Please use the official releases. v0.2 should be out tonight.

File: ansible/roles/docker/defaults/main.yml [hunk 2]
```diff
@@ -2,7 +2,8 @@
 # Default values for docker role
 
 docker_api_port: 2385
-docker_version: 1.9.1
+docker_version: 1.11.1
```
Comment (erikh) [inline ansible/roles/docker/defaults/main.yml]: 1.11.2 is out fwiw

**Comment by erikh** [comment]
I know these are vendored in but I think the `contiv_storage` role is broken atm because I pruned the release tags for nightlies.

If you still can, use v0.1.1.

**Comment by mapuri** [comment]
@erikh 

yes, these are just vendored changes. They shouldn't affect volplugin sanities. We can bump up conitv_storage version independently in ansible/master. Do you want to send a PR? Or I can create one there in a bit

**Comment by mapuri** [comment]
also for docker version I think may be we should bump it up in volplugin's Vagrantfile first, as changing it in ansible will affect netplugin too.

**Comment by mapuri** [comment]
I created https://github.com/contiv/ansible/pull/246 to bump up contiv_storage and other contiv versions

**Comment by erikh** [comment]
@mapuri since I'm out today, can you merge this after tests pass?

**Comment by mapuri** [comment]
merging this now as sanity has passed.

Thanks @erikh @dseevr !

**Pull request merged**